tests/testsuite/help.rs

```diff
@@ -162,8 +166,12 @@ fn help_alias() {
 fn alias_z_flag_help() {
     for cmd in ["build", "run", "check", "test", "b", "r", "c", "t"] {
         cargo_process(&format!("{cmd} -Z help"))
-            .with_stdout_contains(
-                "    -Z allow-features[..]  Allow *only* the listed unstable features",
+            .with_stdout_data(
+                "\
+...
+    -Z allow-features[..]  Allow *only* the listed unstable features
+...
+",
             )
             .run();
```

Comment:
We also might be able to remove this as well

https://github.com/rust-lang/cargo/blob/master/tests/testsuite/cargo/z_help/stdout.term.svg

Reply:
Thanks. Removed in 3fd3879.